app/helpers/checkouts_helper.rb

```diff
@@ -20,10 +20,18 @@ def render_checkout_due_date(checkout)
   end
 
   def render_renewal_select(checkout)
-    check_box_tag 'renewal_list[]', checkout.item_key, false, class: 'checkbox', multiple: true
+    check_box_tag 'renewal_list[]', checkout.item_key, false,
+                  data: { checkbox_type: 'renewal' }, class: 'checkbox', multiple: true
   end
 
   def render_renew_button
     submit_tag 'Renew', class: 'btn btn-primary btn-renewable-submit'
   end
+
+  def render_renew_all
```

Comment:
So, for Holds, I didn't make a special helper for the select all special checkbox. Should I have?

Reply:
hmm I do not think that you have to, but maybe we can chat about if we can refactor to reuse select all stuff in both checkouts and holds?

Reply:
Sounds good